src/sim.ts

```diff
@@ -1,13 +1,17 @@
-import * as Vis from './vis';
-import * as Plane from './plane';
+import {Vis} from './vis';
+import {Plane} from './plane';
 
 export class Sim {
-  thisVis: Vis.Vis;
-  mainPlane: plane.Plane;
+  thisVis: Vis;
```

Comment:
This should be named `vis`, not `thisVis`.

There's no reason to name something `thisX` -- in a method, if you have a parameter `x` and a class property `x`, "`x`" will refer to the parameter and "`this.x`" will refer to the class property.

Here's a demo of that: [TypeScript demo](https://www.typescriptlang.org/play?#code/MYGwhgzhAECiAeYC2AHEBTaBvAUNf0YAXNAHYCuSARugE7QC80ADANw54GQQCWA5qQAUxMpRq0AlNk4ECwAPakI8jADoQ8voIDkABQBKsALQBBAMpmAkgHEAcgFlYtgCraJ7WbIVKV6dZsEAAxQwWmR0ABc6QhIAEiwwAF9A909oAHp06DMAC3lyEAATaBpoAEZVGU9vZTUNLWDaeRQ6CIBPfBF4iJyeCFUklNYMrNz8opLMZkq0-B6+gcZCD1ma338tPQB5M2dTCxsHJ1d3Kq9FWr96oJCwpEjoroTk1M9M7LyC4tKKs7kL9bXRrNVodGLQbq9fqDVLvMZfSblGYERI4VE4GoRaCYJikdAAdzgiFQGEEpz83H4QjK7iAA)